resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/KubernetesV2Suite.scala

```diff
@@ -110,6 +110,19 @@ private[spark] class KubernetesV2Suite extends SparkFunSuite with BeforeAndAfter
     runSparkAppAndVerifyCompletion(SparkLauncher.NO_RESOURCE)
   }
 
+  test("Use client key and client cert file when requesting executors") {
+    sparkConf.setJars(Seq(
+      KubernetesSuite.CONTAINER_LOCAL_MAIN_APP_RESOURCE,
+      KubernetesSuite.CONTAINER_LOCAL_HELPER_JAR_PATH))
+    sparkConf.set(KUBERNETES_DRIVER_CLIENT_KEY_FILE,
+      kubernetesTestComponents.clientConfig.getClientKeyFile)
+    sparkConf.set(KUBERNETES_DRIVER_CLIENT_CERT_FILE,
+      kubernetesTestComponents.clientConfig.getClientCertFile)
+    sparkConf.set(KUBERNETES_DRIVER_CA_CERT_FILE,
+      kubernetesTestComponents.clientConfig.getCaCertFile)
+    runSparkAppAndVerifyCompletion(SparkLauncher.NO_RESOURCE)
```

Comment:
this verifies that the spark app finishes when these confs are set, but I'm not sure it verifies that the key/cert files are used when requesting (e.g. this conf could just be ignored and the test would still pass).  How can we test those?